common/app/staticpages/StaticPages.scala

```diff
@@ -19,7 +19,7 @@ object StaticPages {
       section = Option(SectionSummary(id="email-signup-page", activeBrandings=None)),
       webTitle = webTitle,
       analyticsName = "email-signup-page",
-      contentType = "signup",
+      contentType = "Signup",
```

Comment:
why is this now capitalised? I would've thought to avoid tripping people up we should keep it consistent and `"survey"` above isn't capitalised